### DISCUSSION TITLE
model update files

### DISCUSSION THREAD

**Discussion by helenaherand** [SysBioChalmers/Human-GEM#406] [Q&A] [answered]
How do you update the model files?

**Comment by mihai-sysbio** [comment, marked as answer]
Thanks for question @helenaherand, it mostly depends on what kind of updates you are thinking of.

For curation-type work, the process is described in the [Contributing guidelines](https://github.com/SysBioChalmers/Human-GEM/blob/main/.github/CONTRIBUTING.md#contributing-to-the-modelannotation). This will affect only the `yml` format, which is the single model format that exists in the `develop` branch.

The other model file formats get updated only with new releases, for which there is a [guide on the Wiki](https://github.com/SysBioChalmers/Human-GEM/wiki/How-to-make-a-new-release).

**Comment by JonathanRob** [comment]
I'll convert this to a Discussion, since it seems a better fit.
